### ISSUE TITLE
[suggestion] Wiki/Information about the mod

### ISSUE THREAD

**Issue by milkev** [Draylar/go-fish#4]
so... i know your stance on fishing, i know im supposed to be able to fish in the nether and the end and it has unique mechanics...
but thats about it.
what do the different rods do? blaze rod is presumably the only one that can fish in lava, and diamond presumably just has higher durability, but what about the matrix? the eye of fishing? any of the others?
how do i fish in the end? water doesnt seem to work with any of the rods.
what drops can i even get aside from the fish i can see in rei? i havent really gotten much of note in my creative testing.

**Comment by JoJazzy** [comment]
I agree, the wiki should be expanded and provide some more info to clarify and remove eventual confusion, especially with enchants
